.gitignore

```diff
@@ -30,7 +30,8 @@ __pycache__/*
 
 # Unittest and coverage
 htmlcov/*
-.coverage
```

Comment:
please do not put the configuration file of coverage into the .gitignore (or I don't get why you do that)